airstack/client.go

```diff
@@ -0,0 +1,59 @@
+package airstack
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	gqlClient "github.com/Khan/genqlient/graphql"
+)
+
+// Constants
+const (
+	apiTimeout       = 60 * time.Second
```

Comment:
Is this timeout for query? If so, looks too high.

Reply:
Reduced to 10 sec